From b2810dff807d61cbdd3db906b7522c006b9c7acd Mon Sep 17 00:00:00 2001
From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Date: Fri, 25 Sep 2020 10:04:42 +0300
Subject: [PATCH] bl2: change bl2 location to dram

This changes location of BL2 in DRAM instead RTSRAM due to V3H ws2.0
RTSRAM access issue in V3H ws2.0

Signed-off-by: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
---
 plat/renesas/rcar/include/platform_def.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/plat/renesas/rcar/include/platform_def.h b/plat/renesas/rcar/include/platform_def.h
index c461b95..019c1a8 100644
--- a/plat/renesas/rcar/include/platform_def.h
+++ b/plat/renesas/rcar/include/platform_def.h
@@ -110,9 +110,15 @@
 #endif
 
 #if RCAR_LSI == RCAR_V3H
+#if 0
 #define BL2_BASE			U(0xEB244000)
 #define BL2_LIMIT			U(0xEB264000)
 #define BL2_IMAGE_LIMIT			U(0xEB26E800)
+#else
+#define BL2_BASE			U(0x46000000)
+#define BL2_LIMIT			(BL2_BASE + U(0x100000))
+#define BL2_IMAGE_LIMIT			(BL2_LIMIT + U(0xA800))
+#endif
 #elif RCAR_LSI == RCAR_V3M
 #define BL2_BASE			U(0xE6344000)
 #define BL2_LIMIT			U(0xE6360000)
-- 
2.7.4

